### PR TITLE
Support 'crop to shape' for rotated shapes #971

### LINKED ISSUE #971: Support 'crop to shape' for rotated shapes
Now that the resize lab can redefine the bounding box, we should be able to support crop-to-shape for rotated shapes.

### PR DESCRIPTION
**Pull request by fionachang**
Fixes #971

### CODE REVIEW

**Comment by damithc** [comment]
On a related note, does `crop to shape` work if part of the shape is outside the slide?

**Comment by fionachang** [comment]
Yes

File: PowerPointLabs/PowerPointLabs/CropToShape.cs [hunk 1]
```diff
@@ -60,10 +56,27 @@ public class CropToShape
...
             {
                 if (!VerifyIsShapeRangeValid(shapeRange, handleError)) return null;
+                
+                shapeRange.Cut();
+                shapeRange = PowerPointCurrentPresentationInfo.CurrentSlide.Shapes.Paste();
+                TakeScreenshotProxy(shapeRange);
+
+                var ungroupedRange = UngroupAllForShapeRange(shapeRange);
+                PowerPoint.Shape filledShape = null;
+                var shapes = new string[ungroupedRange.Count];
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/CropToShape.cs]: naming shapeNames?

Comment (kai33) [inline PowerPointLabs/PowerPointLabs/CropToShape.cs]: I'm fine with the rest of codes

**Pull request merged**